spec/models/post_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_length_of(:title).is_at_most(250) }
+  it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
```

Comment:
- [ ] Looking at the screenshot i added to the previous comment, you can see that these tests are failing also, I'll suggest you look in to these your assertions as they seem not to be written correctly. Also as stated in the project requirement, you are meant to test all the methods in the Post model along side the validations. I'll suggest you look into this. here's a suggestion of how a validadton test and a method test for this model should look like. :+1: :wave: :car: 

```ruby

RSpec.describe Post, type: :model do
  subject { Post.new(author: User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', posts_counter: 0), title: 'Hello', text: 'This is my first post', likes_counter: 0, comments_counter: 0) }

  before { subject.save }

  it 'comments_counter must be a number' do
    subject.comments_counter = 'a'
    expect(subject).to_not be_valid
  end

  it 'five_most_recent_comments should return a right result' do
    subject.comments.create(text: 'This is my first comment')
    subject.comments.create(text: 'This is my second comment')
    subject.comments.create(text: 'This is my third comment')
    subject.comments.create(text: 'This is my fourth comment')
    subject.comments.create(text: 'This is my fifth comment')
    subject.comments.create(text: 'This is my sixth comment')
    subject.comments.create(text: 'This is my seventh comment')
    expect(subject.five_most_recent_comments.count).to eq(5)
  end
end
```